susemanager-utils/susemanager-sls/salt/bootstrap/init.sls

```diff
@@ -136,11 +136,16 @@ mgr_update_basic_pkgs:
   pkg.latest:
     - pkgs:
       - openssl
+{%- if grains['os_family'] == 'Suse' and grains['osrelease'] in ['11.3', '11.4'] %}
```

Comment:
Small nitpick: 11.3 is out of support (even LTSS).

Reply:
Yeah, but we still have SLEPOS 11.3 (for retail) and despite we are not going to migrate this to GMC4 (don't worry @mcalmer), I am not sure what will happen for the next version 4.1, 5.0, or whatever is going to be :-(

That's the reason of having it.

Reply:
11.3 is still supported for retail as @juliogonzalez said. And we do best effort support even for older distros. I wonder if we should just check for '11' to catch all?

Reply:
My bad, I did not know about Retail. Good to go :+1: